xml/System.Text.Json/JsonElement.xml

```diff
@@ -436,9 +488,14 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Gets a string representation for the current value appropriate to the value type.</summary>
+        <returns>A string representation for the current value appropriate to the value type.</returns>
+        <remarks>For <see cref="F:System.Text.Json.JsonValueType.Null" />, <see cref="F:System.String.Empty" /> is returned.
```

Comment:
This should be inside a <format> element and use xrefs with `?displayProperty=nameWithType`.

Reply:
```suggestion
        <remarks><format type="text/markdown"><![CDATA[
For <xref:System.Text.Json.JsonValueType.Null?displayProperty=nameWithType>, <xref:System.String.Empty?displayProperty=nameWithType> is returned.
```